modules/soc_evcc/main.sh

```diff
@@ -72,7 +74,11 @@ incrementTimer(){
 getAndWriteSoc(){
 	openwbDebugLog ${DMOD} 0 "Lp$CHARGEPOINT: Requesting SoC"
 	echo 0 > $soctimerfile
-	answer=$($MODULEDIR/../soc_evcc/soc $fztype --user "$username" --password "$password" --vin "$vin" --token "$token" 2>&1)
+	options="${fztype} --user ${username} --password ${password} --vin ${vin} --token ${token}"
+	if [ "${fztype}" == "fiat" ]; then
+		options="${options} --pin ${pin}"
+	fi
+	answer=$($MODULEDIR/../soc_evcc/soc $options 2>&1)
```

Comment:
Es ist etwas sinnfrei, in das übergeordnete Verzeichnis zu wechseln, um dann wieder zurückzugeben.
```suggestion
	answer=$($MODULEDIR/soc $options 2>&1)
```

Reply:
Das war im Original schon so, wurde also offensichtlich vorher in einem Review nicht bemängelt:
```
answer=$($MODULEDIR/../soc_evcc/soc $fztype --user "$username" --password "$password" --vin "$vin" --token "$token" 2>&1)
```

Aber ja, das sehe ich auch so. Habe die Änderungsvorschläge übernommen, Danke!